src/content/i18n/it.yml

```diff
@@ -0,0 +1,88 @@
+rightSidebar.contribute: Contribuisci
+rightSidebar.translatePage: Traduci questa pagina
+# Footer
+footer.privacyPolicy: Privacy Policy
+# Used in `<Since>`: Added in: v0.24.0 [NEW]
+since.addedIn: 'Aggiunto in:'
+since.new: Nuovo
+since.beta: Beta
+# Installation Guide
+install.autoTab: CLI Automatica
+install.manualTab: Setup Manuale
+# `<DeployGuidesNav>` vocabulary
+deploy.sectionTitle: Guide di Pubblicazione
+deploy.altSectionTitle: Altre Guide di Pubblicazione
+deploy.ssrTag: SSR
+deploy.staticTag: Statica
+# CMS Guides vocabulary
+cms.navTitle: Altre guide per le CMS
+# Migration Guides vocabulary
+migration.navTitle: Altre guide per le migrazioni
+# `<RecipeLinks>` vocabulary
+recipesLink.singular: 'Soluzione simile:'
+recipesLink.plural: Soluzioni simili
+# 404 Page
+404.title: Non trovata
+404.content: Questa pagina non si trova nel nostro sistema solare.
+404.linkText: Torna alla home.
+# Integrations vocabulary
+integrations.changelog: Registro delle modifiche
+integrations.footerTitle: Altre Integrazioni
+integrations.renderers: Framework UI
+integrations.adapters: Adattatori SSR
+integrations.others: Altre Integrazioni
+integrations.more: Più integrazioni
+# Checklist component
+checklist.or: oppure
+# Multiple Choice component
+multipleChoice.defaultCorrect: Corretto!
+multipleChoice.defaultIncorrect: Riprova!
+multipleChoice.submitLabel: Invia
+# Tutorial Progress
+progress.todo: Da fare
+progress.done: Completato
+# Tutorial Navigation
+tutorial.trackerLabel: Progresso del Tutorial
+tutorial.unit: Unità
+# Tutorial
+tutorial.getReady: Preparati a…
+# Feedback Fish widget
+feedback.button: Dicci che ne pensi
+feedback.a11yLabel: Modulo per il feedback
+feedback.formTitle: A cosa stai pensando?
+feedback.categoryGroupLabel: Scegli la categoria del feedback
+feedback.issue: Problema
+feedback.createIssue: Crea una Issue su GitHub
+feedback.createIssue.description: Il modo più rapido per segnalare un problema al nostro team.
+feedback.sendFeedback: Inviateci un feedback
+feedback.sendFeedback.description: Inviateci direttamente un messaggio.
+feedback.idea: Idea
+feedback.other: Altro
+feedback.messageA11yLabel: Messaggio
+feedback.placeholder: Cosa vuoi farci sapere?
+feedback.submit: Invia il feedback
+feedback.close: Chiudi il modulo per il feedback
+feedback.success: Grazie! Abbiamo ricevuto il tuo feedback.
+# Code snippet vocabulary
+expressiveCode.terminalWindowFallbackTitle: Finestra del terminale
+expressiveCode.copyButtonTooltip: Copia
+expressiveCode.copyButtonCopied: Copiato!
+# Backend Guides vocabulary
+backend.navTitle: Altre guide per servizi backend
+# Starlight banner
+starlight.title: Vuoi creare la tua Documentazione?
+starlight.description: Usa questo template per iniziare.
+# `<StudioHeading>` component
+studioHeading.label: Funzionalità Studio
+# DocSearch component strings
+docsearch.button: Cerca
+docsearch.shortcutLabel: Premi / per cercare
+docsearch.placeholder: Cerca nella documentazione
```

Comment:
```suggestion
a11y.sectionLink: Sezione intitolata
# Site settings
site.title: Documentazione di Astro
# Left Sidebar
leftSidebar.sponsoredBy: Sponsorizzato da
# Right Sidebar
rightSidebar.community: Comunità
rightSidebar.joinDiscord: Unisciti a noi su Discord
rightSidebar.contribute: Contribuisci
rightSidebar.translatePage: Traduci questa pagina
# Footer
footer.privacyPolicy: Privacy Policy
# Used in `<Since>`: Added in: v0.24.0 [NEW]
since.addedIn: 'Aggiunto in:'
since.new: Nuovo
since.beta: Beta
# Installation Guide
install.autoTab: CLI Automatica
install.manualTab: Setup Manuale
# `<DeployGuidesNav>` vocabulary
deploy.sectionTitle: Guide di Pubblicazione
deploy.altSectionTitle: Altre Guide di Pubblicazione
deploy.ssrTag: SSR
deploy.staticTag: Statica
# CMS Guides vocabulary
cms.navTitle: Altre guide per i CMS
# Media Guides vocabulary
media.navTitle: Altre guide per i DAM
# Migration Guides vocabulary
migration.navTitle: Altre guide per le migrazioni
# `<RecipeLinks>` vocabulary
recipesLink.singular: 'Soluzione simile:'
recipesLink.plural: Soluzioni simili
# 404 Page
404.title: Non trovata
404.content: Questa pagina non si trova nel nostro sistema solare.
404.linkText: Torna alla home.
# Integrations vocabulary
integrations.changelog: Registro delle modifiche
integrations.footerTitle: Altre Integrazioni
integrations.renderers: Framework UI
integrations.adapters: Adattatori SSR
integrations.others: Altre Integrazioni
integrations.more: Più integrazioni
# Checklist component
checklist.or: oppure
# Multiple Choice component
multipleChoice.defaultCorrect: Corretto!
multipleChoice.defaultIncorrect: Riprova!
multipleChoice.submitLabel: Invia
# Tutorial Progress
progress.todo: Da fare
progress.done: Completato
# Tutorial Navigation
tutorial.trackerLabel: Progresso del Tutorial
tutorial.unit: Unità
# Tutorial
tutorial.getReady: Preparati a…
# Feedback Fish widget
feedback.button: Dicci che ne pensi
feedback.a11yLabel: Modulo per il feedback
feedback.formTitle: A cosa stai pensando?
feedback.categoryGroupLabel: Scegli la categoria del feedback
feedback.issue: Problema
feedback.createIssue: Crea una Issue su GitHub
feedback.createIssue.description: Il modo più rapido per segnalare un problema al nostro team.
feedback.sendFeedback: Inviateci un feedback
feedback.sendFeedback.description: Inviateci direttamente un messaggio.
feedback.idea: Idea
feedback.other: Altro
feedback.messageA11yLabel: Messaggio
feedback.placeholder: Cosa vuoi farci sapere?
feedback.submit: Invia il feedback
feedback.close: Chiudi il modulo per il feedback
feedback.success: Grazie! Abbiamo ricevuto il tuo feedback.
# Code snippet vocabulary
expressiveCode.terminalWindowFallbackTitle: Finestra del terminale
expressiveCode.copyButtonTooltip: Copia
expressiveCode.copyButtonCopied: Copiato!
# Backend Guides vocabulary
backend.navTitle: Altre guide per servizi backend
# Starlight banner
starlight.title: Vuoi creare la tua Documentazione?
starlight.description: Usa questo template per iniziare.
# `<StudioHeading>` component
studioHeading.label: Funzionalità Studio

# DocSearch component strings
docsearch.button: Cerca
docsearch.shortcutLabel: Premi / per cercare
docsearch.placeholder: Cerca nella documentazione
# SEARCH BOX
docsearch.modal.searchBox.resetButtonTitle: Cancella la ricerca
docsearch.modal.searchBox.resetButtonAriaLabel: Cancella la ricerca
docsearch.modal.searchBox.cancelButtonText: Annulla
docsearch.modal.searchBox.cancelButtonAriaLabel: Annulla
docsearch.modal.searchBox.searchInputLabel: Cerca
# START SCREEN
docsearch.modal.startScreen.recentSearchesTitle: Recenti
docsearch.modal.startScreen.noRecentSearchesText: Nessuna ricerca recente
docsearch.modal.startScreen.saveRecentSearchButtonTitle: Salva questa ricerca
docsearch.modal.startScreen.removeRecentSearchButtonTitle: Rimuovi questa ricerca dalla cronologia
docsearch.modal.startScreen.favoriteSearchesTitle: Preferiti
docsearch.modal.startScreen.removeFavoriteSearchButtonTitle: Rimuovi questa ricerca dai preferiti
# ERROR SCREEN
docsearch.modal.errorScreen.titleText: Impossibile recuperare i risultati
docsearch.modal.errorScreen.helpText: Prova a controllare la tua connessione di rete.
# FOOTER
docsearch.modal.footer.selectText: per selezionare
docsearch.modal.footer.selectKeyAriaLabel: Tasto Invio
docsearch.modal.footer.navigateText: per navigare
docsearch.modal.footer.navigateUpKeyAriaLabel: Freccia su
docsearch.modal.footer.navigateDownKeyAriaLabel: Freccia giù
docsearch.modal.footer.closeText: per chiudere
docsearch.modal.footer.closeKeyAriaLabel: Tasto Esc
docsearch.modal.footer.searchByText: Cerca con
# NO RESULTS SCREEN
docsearch.modal.noResultsScreen.noResultsText: Nessun risultato per
docsearch.modal.noResultsScreen.suggestedQueryText: Prova a cercare
docsearch.modal.noResultsScreen.reportMissingResultsText: Pensi che questa ricerca dovrebbe avere dei risultati?
docsearch.modal.noResultsScreen.reportMissingResultsLinkText: Faccelo sapere.
```

Reply:
Thanks for the update @casungo! I used [Diffchecker](https://www.diffchecker.com/text-compare/) to see what was added between the two versions more easily.